src/tools/rustdoc-gui/tester.js

```diff
@@ -153,7 +167,11 @@ async function main(argv) {
     files.sort();
 
     console.log(`Running ${files.length} rustdoc-gui tests...`);
-    process.setMaxListeners(files.length + 1);
+    if (opts["jobs"] < 1) {
+        process.setMaxListeners(files.length + 1);
+    } else {
+        process.setMaxListeners(opts["jobs"]);
```

Comment:
Will this give a warning to the users? If I remember correctly that warning was the reason why I increased the number to `files.length + 1`

Reply:
I didn't have any warning, so no idea. But we shouldn't go above the limit provided by the tester in any case.

Reply:
Cool :relaxed: It is a very clean solution :+1:

Reply:
\o/